libevm/VMFactory.cpp

```diff
@@ -63,6 +71,27 @@ VMKindTableEntry vmKindsTable[] = {
 #endif
 };
 
+evmc_instance* createDllEVMC(const fs::path path)
+{
+    static auto createFn = [&] {
+        auto symbols = dll::library_info{path}.symbols();
+        auto it = std::find_if(symbols.begin(), symbols.end(),
+            [](const std::string& symbol) { return symbol.find("evmc_create_") == 0; });
```

Comment:
Will this stop on the first match?

Should this rather throw an error if there are multiple matches? (So it is consistent with what's loaded.)

Reply:
Yes, it stops. To be consistent with what?
I would keep it simple. Still you don't have any guarantees that the function will be ABI compatible and everything will not explode.

Reply:
If there are two symbols with the `evmc_create_` prefix is it consistent which one will be picked up across platforms? However, if there are multiple matches I'd rather just bail out.

Reply:
Addressed.

Reply:
Thanks!